nacc/uds3/filters.py

```diff
@@ -223,3 +231,12 @@ def filter_csv_vnum(Ptid, visit_num, record):
 def filter_csv_ptid(Ptid, record):
     if record['ptid'] == Ptid:
         return record
+
+def test_all_ptid(ptid_list, record):  
+    for x in ptid_list:
+        if record == x:
+            return True
```

Comment:
You can do this more simply (and more Pythonically) with:
```
if record in ptid_list:
    return True
```

In fact, this doesn't even need to be a function. You can change line 140 and 142 to simply checking `if record in ptid_list` (of course, you'll have to switch out 'ptid_list' with whatever list you are checking).